lib/ansible/plugins/connection/ssh.py

```diff
@@ -610,24 +599,6 @@ def _create_control_path(
         cpath = '%(directory)s/' + digest[:10]
         return cpath
 
-    @staticmethod
-    def _sshpass_available() -> bool:
-        global SSHPASS_AVAILABLE
```

Comment:
if we restore the sshpass code, a better 'available' test would be to use the get_bin_path function